scripts/format-code

```diff
@@ -159,6 +159,7 @@ check_clang-format()
             echo "clang-format is not installed. Please run scripts/install-prereqs first."
             exit 1
         fi
+        cf="clang-format"
```

Comment:
I see from the commit message that this change was necessary because the path found on Windows has a space in it. Can we fix this instead by quoting uses of `cf` as appropriate below? Otherwise I feel someone is going to just delete this line as it doesn't make sense out of context.

Reply:
I actually became aware of an issue yesterday that format-code is no longer working properly. I am investigating...

Reply:
(And so would rather not touch it for now.)

Reply:
Oh, and part of that issue is that you can have both clang-format and clang-format-7 installed, and clang-format will actually point to clang-format-3.8, so I don't think this approach will work correctly.

Reply:
Roger that. I'll hold off from any changes for the time being 😃

Reply:
Thanks @dragoshsch. The changes I was working on went in, so you're now free to rebase.

However, I think we still need to go with the proper quoting approach because `clang-format` will point to clang-format-3.8 on Ubuntu 16.04 since it's a meta-package that points to the "stable" version.